app/assets/javascripts/channels/comment.coffee

```diff
@@ -0,0 +1,53 @@
+App.comment = App.cable.subscriptions.create "CommentChannel",
+  connected: ->
+    # Called when the subscription is ready for use on the server
+
+  disconnected: ->
+    # Called when the subscription has been terminated by the server
+
+
+  # Called when there's incoming data on the websocket for this channel
+  received: (data) ->
+    event = new CommentEvent(data)
+    event.process() if event.isRelevant()
+
+
+class CommentEvent
+  constructor: (data) ->
+    @$feed     = $("[data-commentable='#{data.comment_feed_id}']")
+    @action    = data.action
+    @commentId = data.comment_id
+    @html      = data.html
+
+  # We need to make sure that the comment we've been notified about belongs
+  # on the currently-viewed page
+  isRelevant: ->
+    @$feed.length != 0
+
+  toggleActions: ->
+    $comment = $("#comment_#{@commentId}")
+    unless $('body').data('currentUserId') == $comment.data('userId')
+      $comment.find('.actions, .edit_comment').remove()
+
+  updateCounters: ->
+    count = $(".comments-list div.comment").length
+    $("#comments_feed_no_comments").toggle(count == 0)
+    $("#comments-count-badge").text(count)
+
+  process: ->
```

Comment:
Idea: remove custom js to update a specific page and reload the page with `TurboLinks` when `ActionCable` says so

Reply:
If we're going to reload the entire page, what's the point in using ActionCable? We might as well just stick with polling since the JS would be simple (essentially, `if serverSaysWeMustRefresh then refresh()`) and ActionCable is big overhead.

Plus it would be very jarring to hit people with a full-page refresh that happens at a point they have no control over (i.e. when someone else comments).